demisto_sdk/commands/common/update_id_set.py

```diff
@@ -2202,3 +2282,34 @@ def is_same_source(source1, source2) -> bool:
 def sort(data):
     data.sort(key=lambda r: list(r.keys())[0].lower())  # Sort data by key value
     return data
+
+
+def update_excluded_items_dict(excluded_items_from_id_set: dict, excluded_items_to_add: dict,
```

Comment:
[Target before values](https://mikelyons.org/2020/05/13/coding-standards-argument-ordering):
since you are updating both `excluded_items_from_id_set` and `excluded_items_by_type` with `excluded_items_to_add` it makes more sense to have them next to each other and move the `excluded_items_to_add` argument to the end.